zephyr/include/rtos/wait.h

```diff
@@ -12,32 +12,31 @@
 #include <stdbool.h>
 #include <zephyr/kernel.h>
 
-/* TODO: use equivalent Zephyr */
 static inline void idelay(int n)
 {
-	while (n--)
-		asm volatile("nop");
+	k_busy_wait((uint32_t)n);
 }
 
 /* DSP default delay in cycles - all platforms use this today */
 #define PLATFORM_DEFAULT_DELAY	12
```

Comment:
Can we get rid of PLATFORM_DEFAULT_DELAY now?

Reply:
Yes, this one can be removed.